database/001_ddl_create_tables.sql

```diff
@@ -1,46 +1,42 @@
 CREATE TYPE user_role_type as ENUM ('customer', 'stock_manager');
-
 CREATE TYPE order_status_type as ENUM (
-    'created', 'pending', 'paid', 'separation', 'delivered', 'canceled'
+    'created',
+    'pending',
+    'paid',
+    'separation',
+    'delivered',
+    'canceled'
 );
-
 CREATE TYPE product_size_enum AS ENUM ('P', 'M', 'G');
-
 CREATE TYPE payment_status_type as ENUM ('pending', 'paid', 'canceled');
-
 CREATE TABLE users (
-    user_id SERIAL PRIMARY KEY,
-    username VARCHAR(50) NOT NULL,
-    password_hash VARCHAR(50) NOT NULL,
-    email VARCHAR(50),
+    user_id VARCHAR(128) PRIMARY KEY,
+    password_hash VARCHAR(64) NOT NULL,
```

Comment:
Since all password logic is handled by firebase we can remo password has from users table